stable/cloudserver/Chart.yaml

```diff
@@ -13,4 +13,6 @@ maintainers:
     email: giacomoguiulfo@gmail.com
```

Comment:
I don't think he wants to be maintaining this any more. You can just remove his name from both.